src/components/AChat/AChat.vue

```diff
@@ -78,128 +78,110 @@ export default {
     }
   },
   emits: ['scroll', 'scroll:bottom', 'scroll:top'],
```

Comment:
Can we type the `emits`? Currently defined as `any`.

![image](https://github.com/user-attachments/assets/4fe99398-0841-4b3e-b3c4-0920f64bc14e)